spec/editors_draft.html

```diff
@@ -3298,6 +3298,231 @@ <h2>Evaluation Semantics</h2>
+      <pre id="load-renaming-blank-example-load-twice"
+         title="Repeated loading of Composite Values with blank node"
+         data-transform="updateExample"
+         data-content-type="application/sparql-update"
+         class="nohighlight example">
+      <!--
+      INSERT DATA { <s> <p2> "[ _:b0 ]"^^cdt:List } ;
+      INSERT DATA { <s> <p2> "[ _:b0 ]"^^cdt:List } ;
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-load-twice-result"
+         title="Repeated loading of Composite Values with blank node: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      <http://example.org/s> <http://example.org/p2> "[ _:b002 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      <http://example.org/s> <http://example.org/p2> "[ _:b003 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>Loading the Turtle in <a href="#load-renaming-blank-example-consistent-ids"></a> with a triple with `_:b0` appearing in both the subject position and inside a <a>cdt:List literal</a> object MUST result in a dataset with a single triple whose subject is the same blank node as contained in the single-element <a>cdt:List literal</a> object, and where that blank node MUST NOT have the identifier "b0" (<a href="#load-renaming-blank-example-consistent-ids-result"></a>):</p>
+      <pre id="load-renaming-blank-example-consistent-ids"
+         title="Producing consistent blank node identifiers shared between RDF and Composite Values"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      _:b0 ex:p3 "[ _:b0 ]"^^cdt:List .
+      -->
+      </pre>
+      <pre id="load-renaming-blank-example-consistent-ids-result"
+         title="Producing consistent blank node identifiers shared between RDF and Composite Values: Result"
+         data-transform="updateExample"
+         data-content-type="text/n-triples"
+         class="nohighlight example">
+      <!--
+      _:b004 <http://example.org/p3> "[ _:b004 ]"^^<http://w3id.org/awslabs/neptune/SPARQL-CDTs/List> .
+      -->
+      </pre>
+
+      <p>Loading the Turtle in <a href="#load-renaming-blank-reuse"></a> with a triple with a <a>cdt:Map literal</a> containing two map entries, both having the same blank node value, MUST result in a dataset with a single triple whose <a>cdt:Map literal</a> object contains two map entries having the same blank node value, and whose identifier MUST NOT be "b0" (<a href="#load-renaming-blank-reuse-result"></a>):</p>
```

Comment:
The mention of "blank node value" got me confused, until I realized that you probably want to say "blank node as their value", right?

Reply:
Yes. I will update the text to read:

> Loading the Turtle in Example 50 with a triple with a `cdt:Map` literal containing two map entries, both having the same blank node as their value

Reply:
Here is the update as a directly-applicable commit (unless you have already implemented this change in your local copy):
```suggestion
      <p>Loading the Turtle in <a href="#load-renaming-blank-reuse"></a> with a triple with a <a>cdt:Map literal</a> containing two map entries, both having the same blank node as their value, MUST result in a dataset with a single triple whose <a>cdt:Map literal</a> object contains two map entries having the same blank node value, and whose identifier MUST NOT be "b0" (<a href="#load-renaming-blank-reuse-result"></a>):</p>
```